tests/driver_enc28j60/Makefile.ci

```diff
@@ -1,35 +1,16 @@
 BOARD_INSUFFICIENT_MEMORY := \
     arduino-duemilanove \
     arduino-leonardo \
-    arduino-mega2560 \
     arduino-nano \
     arduino-uno \
-    atmega1284p \
     atmega328p \
     atmega328p-xplained-mini \
-    atxmega-a3bu-xplained \
-    bluepill-stm32f030c8 \
-    derfmega128 \
-    i-nucleo-lrwan1 \
-    mega-xplained \
-    microduino-corerf \
-    msb-430 \
-    msb-430h \
+    esp8266-esp-12x \
+    esp8266-olimex-mod \
+    esp8266-sparkfun-thing \
```

Comment:
That's odd. Maybe the linking issue is not due to memory limitation, but has other causes? If I recall correctly, even the old generation ESPs are pretty well equipped with RAM and flash.